Makefile

```diff
@@ -11,4 +11,8 @@ endif
 build: custom_ca
 	docker build . -t local/hive:$(branch)
 	docker tag  local/hive:$(branch) local/hive:latest
+up:
+	vagrant up --provision
```

Comment:
-- Is this what we want? vagrant up should be sufficient. --provision will cause it to always run the provisioning.

Reply:
Yes. 

ps. I do not want to mix issue #4 in this PR. Next PR I will focus on CD/CI and how to separate provision by some input for local `setup` or `test`.

Reply:
yup